app/controllers/Application.scala

```diff
@@ -42,6 +43,7 @@ class Application(
     val redirectUrl = request.fastlyCountry match {
       case Some(UK) => "/uk"
       case Some(US) => "/us"
+      case Some(Europe)  => "/eu"
```

Comment:
Will this result in a double redirect?

`/` -> `/eu` -> `/eu/contribute`

Reply:
Yes, it is the same case for `us`